### PR TITLE
Fix for link when there is no project in travis.

### PR DESCRIPTION
**Pull request by ramen74**
Cleaner look when the pivotal link is broken.

### CODE REVIEW

**Comment by kelvintaywl** [comment]
LGTM, save for the test fixes :hamburger:

File: plugins/travis/travis.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (c TravisColumn) RenderDetail() (template.HTML, error) {
 		url = fmt.Sprintf("%s/%s/%s", rootUrls[1], c.Organization, c.Project)
 		svg = fmt.Sprintf("%s/%s/%s.svg?token=%s&branch=master", rootUrls[1], c.Organization, c.Project, c.Token)
 	}
-	return template.HTML(fmt.Sprintf("<td><a target=_blank href=%s><img src=%s></img></a></td>", url, svg)), nil
+	return template.HTML(fmt.Sprintf("<td><a target=_blank href=%s><img src=%s onerror='this.style.display = \"none\"'></img></a></td>", url, svg)), nil
```
Comment (yugui) [inline plugins/travis/travis.go]: Use raw string rather than escaping double quotes.

File: plugins/travis/travis_test.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func TestRenderDetailPublic(t *testing.T) {
 	if err != nil {
 		t.Errorf(err.Error())
 	}
-	want := template.HTML("<td><a target=_blank href=https://travis-ci.org/test/test_public><img src=https://travis-ci.org/test/test_public.svg?branch=master></img></a></td>")
+	want := template.HTML("<td><a target=_blank href=https://travis-ci.org/test/test_public><img src=https://travis-ci.org/test/test_public.svg?branch=master onerror='this.style.display = \"none\"'></img></a></td>")
```
Comment (yugui) [inline plugins/travis/travis_test.go]: ditto.

File: plugins/travis/travis_test.go [hunk 3]
```diff
@@ -100,7 +100,7 @@ func TestRenderDetailPrivate(t *testing.T) {
 	if err != nil {
 		t.Errorf(err.Error())
 	}
-	want := template.HTML("<td><a target=_blank href=https://magnum.travis-ci.com/test/test_private><img src=https://magnum.travis-ci.com/test/test_private.svg?token=test_token&branch=master></img></a></td>")
+	want := template.HTML("<td><a target=_blank href=https://magnum.travis-ci.com/test/test_private><img src=https://magnum.travis-ci.com/test/test_private.svg?token=test_token&branch=master onerror='this.style.display = \"none\"'></img></a></td>")
```
Comment (yugui) [inline plugins/travis/travis_test.go]: ditto.

**Pull request merged**